### ISSUE TITLE
files created by cobalt init needs update

### ISSUE THREAD

**Issue by tigercosmos** [cobalt-org/cobalt.rs#353]
Use latest version. After `cobalt init`, there are some file created.

Then it shows:

```
[warn]   Loading `include`s relative to `source` is deprecated, see _layouts/post.liquid.
```

because it has
```
      {% if is_post %}
        {% include '_layouts/post.liquid' %}
      {% else %}
        {{ content }}
      {% endif %}
```

**Comment by epage** [comment]
Thanks for reporting this!

A fix is already in #346

**Comment by epage** [comment]
Fixed by https://github.com/cobalt-org/cobalt.rs/pull/346
